storage/s3-sdk/README.rst

```diff
@@ -0,0 +1,73 @@
+.. This file is automatically generated. Do not edit this file directly.
+
+Google Cloud Storage Python Samples
+===============================================================================
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=storage/s3-sdk/README.rst
+
+
+This directory contains samples for Google Cloud Storage. `Google Cloud Storage`_ provides support for S3 API users through the GCS XML API.
+Learn more about migrating data from S3 to GCS at https://cloud.google.com/storage/docs/migrating.
+
+
+
+
+.. _Google Cloud Storage: https://cloud.google.com/storage/docs
+
+Setup
+-------------------------------------------------------------------------------
+
+
+Install Dependencies
+++++++++++++++++++++
+
+#. Clone python-docs-samples and change directory to the sample directory you want to use.
+
+    .. code-block:: bash
+
+        $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
```

Comment:
Suggest you add the cd command after the clone.

Reply:
Also, should this tell the user to set the authentication environment variables?

Reply:
I modified the example to pass in credentials using CLI parameter.

I looked at other samples and I didn't see `cd ./../` when generating README's with templates. For consistency, I'm going to keep as is as it impacts other samples.